fe/fe-core/src/main/java/org/apache/doris/common/ThreadPoolManager.java

```diff
@@ -134,6 +134,15 @@ public static ThreadPoolExecutor newDaemonFixedThreadPool(int numThread, int que
                 poolName, needRegisterMetric);
     }
 
+    public static ThreadPoolExecutor newDaemonFixedThreadPool(int numThread, int queueSize,
```

Comment:
could you explain the difference between newDaemonFixedThreadPool and newDaemonFixedPriorityThreadPool? what is been "fixed"

Reply:
thread number